kombu/serialization.py

```diff
@@ -415,18 +415,18 @@ def not_available(*args, **kwargs):
 
 def enable_insecure_serializers(choices=NOTSET):
     """Enable serializers that are considered to be unsafe.
-
     Note:
         Will enable ``pickle``, ``yaml`` and ``msgpack`` by default, but you
         can also specify a list of serializers (by name or content type)
         to enable.
     """
     choices = ['pickle', 'yaml', 'msgpack'] if choices is NOTSET else choices
-    for choice in choices:
```

Comment:
Why not add an early return here?

```python
return if choices is None
```

Reply:
If you have time, can you push suggested edits on my branch?